modules/schema/src/main/java/org/apache/ignite/internal/schema/row/RowAssembler.java

```diff
@@ -799,8 +729,11 @@ private void flush() {
             }
         }
 
+        long[] parts = MurmurHash3.hash128x64(buf.unwrap().array(), KEY_CHUNK_OFFSET, keyChunkLength, 0);
```

Comment:
We use 32-bit hash.
So, why we need to allocate long[] for it ?

Reply:
fixed